v4/source/channels.yaml

```diff
@@ -724,7 +724,9 @@
 
         ##### Permissions
 
-        `manage_team` permission for the team of the channel.
+        `manage_team` permission for the channels team on version < 5.28.
+        `convert_public_channel_to_private` permission for the channel if updating privacy to 'P' on version >= 5.28.
+        `convert_private_channel_to_public` permission for the channel if updating privacy to 'O' on version >= 5.28.
```

Comment:
@reflog Is this okay? to have a different permission requirement on a different version and then document the changes as such? Or should I only show the latest permission requirements? Or does this break versioning? technically any role that had `manage_team` will now get the two new permissions so it shouldnt break any existing usage for anyone

Reply:
I think it's good to document different versions. :+1: